test_scripts/Defects/8_0/2399__SDL_respond_UNSUPPORTED_RESOURCE_in_case_SDL_4_0_was_omited.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2399
+---------------------------------------------------------------------------------------------------
+-- Description:
+-- SDL does respond UNSUPPORTED_RESOURCE to SystemRequest(QUERY_APPS) in case app is registered with protocol version
+-- less than 4.0
+---------------------------------------------------------------------------------------------------
+-- In case:
+-- 1) App is registered and activated
+-- 2) Mobile app requests SystemRequest(QUERY_APPS)
+-- SDL does:
+-- - Respond SystemRequest (UNSUPPORTED_RESOURCE, success:false) to mobile application
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local protocol_version = 3
```

Comment:
@VNikiforov-dev I would propose to use `camelCase` naming style instead `snake_case` here
```
local protocol_version = 3
=>
local protocolVersion = 3
```
And rename this variable according its purpose 
```
protocolVersion => maxSdlProtocolVersion
```